pkgloader-recommended.sty

```diff
@@ -69,6 +69,20 @@
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  A list of recommended rules extracted from the refenums
+%  documentation and code:
+%
+%    \begin{macrocode}
+\Load {refenums} after   {hyperref, cleveref, csquotes,
+                          ifthen}
+                 if loaded
+                 because {refenums relies on these packages}
```

Comment:
Relies on them? If so, does it not just load them itself, in which case they couldn't possibly load out of order?

The main question is: If a document author lists them out of order, will things go wrong? If not, we don't need a pkgloader rule. This goes for the other PRs too. Especially #13, which says "selnolig depends on fontspec".

If this rule still makes sense, please remove the additional spacing after `after` and don't wrap the line.

Reply:
Do you know the famous "Option clash for package" - http://www.tex.ac.uk/FAQ-optionclash.html?

Please try this on your machine:

    \documentclass{article}
    \usepackage{refenums}
    \usepackage[colorlinks=false]{hyperref}

    \begin{document}
    Test
    \end{document}

Result:

```
! LaTeX Error: Option clash for package hyperref.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...

l.4
```

I am aware that `\PassOptionsToPackage` helps here. Did I read "managing the options" too strong at the package description? (emphasis by me)

> A LaTeX package for **managing the options** and loading order of other packages

Reply:
I removed the wrapping. I hope, that an overstanding character is OK for you:

![grafik](https://user-images.githubusercontent.com/1366654/37581035-0586e096-2b47-11e8-8a0f-e791734dec0f.png)

I personally don't like the 80 chars rule, do you want to enforce it?

Reply:
Ah, yes. Forgive me, it's been a number of years since I've written any LaTeX, but I'm caught up again. Yes, the rule is fine, but if the reason is an options clash, then actually use that as the reason. "refenum relies on these packages" is a bit vague.

I don't want to enforce the 80 chars rule. Sorry if I'm being inconsistent with this. It now seems to me that the only reason to limit it to _x_ chars is so that it can fit in a pdf, which is not required for these rules.